modules/ROOT/pages/managing-your-account.adoc

```diff
@@ -19,11 +19,12 @@ A form appears, where you can provide your contact information and create a user
 ** Alphanumeric characters
 ** Dashes
 ** Underscores
-* Password - Must be at least six characters in length and can contain only the following: +
+* Password - Must be at least eight characters in length with at least one uppercase and one lowercase letter. Your password can contain only the following: +
 ** Upper and lower case characters
 ** Numbers
 ** Non-alphanumeric symbols
-* Company - Can be any name unless you were invited to join an existing organization, in which case the company name is already set.
+* Company - Must be at least two characters in length. +
```

Comment:
You have to put the plus sign on it's own line to make it a full paragraph break.

Reply:
That is useful information. The + breaks are a mystery to me. Thank you for explaining that!! 😅 
I think this looks how I intended for now.